src/pocketmine/command/defaults/TitleCommand.php

```diff
@@ -0,0 +1,97 @@
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+namespace pocketmine\command\defaults;
+
+use pocketmine\command\CommandSender;
+use pocketmine\event\TranslationContainer;
+use pocketmine\plugin\Plugin;
+use pocketmine\utils\TextFormat;
+
+class TitleCommand extends VanillaCommand{
+
+	public function __construct($name){
+		parent::__construct(
+			$name,
+			"%pocketmine.command.title.description",
+			"%commands.title.usage"
+		);
+		$this->setPermission("pocketmine.command.title");
+	}
+
+	public function execute(CommandSender $sender, $currentAlias, array $args){
+		if(!$this->testPermission($sender)){
+			return true;
+		}
+
+		if(count($args) < 2){
+		    $sender->sendMessage(new TranslationContainer(TextFormat::RED . "%commands.generic.usage", [$this->usageMessage]));
+		    return true;
+		}
+		
+		$player = $sender->getServer()->getPlayer($args[0]);
+		
+		if($player === null){
+		    $sender->sendMessage(new TranslationContainer(TextFormat::RED . "%commands.generic.player.notFound"));
+		    return true;
+		}
+		
+		if($args[1] === "clear"){
+		    $player->removeTitles();
+		    
+		}elseif($args[1] === "reset"){
+		    $player->resetTitles(); 
+		    
+		}elseif($args[1] === "title"){
+		    if(count($args) < 3){
+		        $sender->sendMessage(new TranslationContainer(TextFormat::RED . "%commands.generic.usage", [$this->usageMessage]));
+		        
+		        return false;
+		    }  
+		    $player->addTitle(implode(" ", array_slice($args, 2)));
+		  
+		}elseif($args[1] === "subtitle"){
+		    if(count($args) < 3){
+		        $sender->sendMessage(new TranslationContainer(TextFormat::RED . "%commands.generic.usage", [$this->usageMessage]));
+		        
+		        return false;
+		    }
+		    $player->addSubTitle(implode(" ", array_slice($args, 2)));
+		}elseif($args[1] === "actionbar"){
+		    if(count($args) < 3){
+		        $sender->sendMessage(new TranslationContainer(TextFormat::RED . "%commands.generic.usage", [$this->usageMessage]));
+		        
+		        return false;
+		    }
+		    $player->addActionBarMessage(implode(" ", array_slice($args, 2)));    
+		}elseif($args[1] === "times"){
+		    if(count($args) !== 5){
+		        $sender->sendMessage(new TranslationContainer(TextFormat::RED . "%commands.generic.usage", [$this->usageMessage]));
+		    
+		        return false;
+		    }
+		    $player->setTitleDuration($this->getInteger($sender, $args[3]), $this->getInteger($sender, $args[4]), $this->getInteger($sender, $args[5]));
```

Comment:
These argument indices are incorrect. Player is 0, target is 1, fade-in should be 2, stay-time 3, fade-out 4. This causes unexpected behaviour when trying to add titles to a player.